packages/react-instantsearch-core/src/connectors/__tests__/connectPoweredBy.js

```diff
@@ -1,6 +1,6 @@
-import connect from './connectPoweredBy';
+import connect from '../connectPoweredBy';
 
-jest.mock('../core/createConnector');
+jest.mock('../../core/createConnector', () => x => x);
```

Comment:
where's x defined?

Reply:
Nowhere it's a function that returns a function.

Reply:
should it not be `() => () => undefined` or `() => () => {}` then?

Reply:
Nope because we use the return value of the connector for the test. The function above takes the connector declaration (the object with `getProvidedProps`, etc...) and return it. This way we can test each function independently. It was already the case, I inline the mock instead of rely on the `__mock__` file.